apps/src/lab2/projects/SourcesStore.ts

```diff
@@ -68,7 +74,7 @@ export class RemoteSourcesStore implements SourcesStore {
     projectType?: ProjectType,
     forceNewVersion = false
   ) {
-    let options = undefined;
+    let options: SaveSourceOptions | UpdateSourceOptions = {projectType};
```

Comment:
Nit: just `SaveSourceOptions` here, then cast to `UpdateSourceOptions` when you assign the other options below.

Reply:
Oh yeah - and updated in `sourcesApi` as well. Thanks!